src/main/scala/com/cognite/sdk/scala/playground/wellDataLayer.scala

```diff
@@ -0,0 +1,200 @@
+package com.cognite.sdk.scala.playground
+
+final case class Source(
+    name: String,
+    description: Option[String] = None
+)
+
+final case class SourceItems(
+    items: Seq[Source]
+)
+
+final case class LimitAndCursor(cursor: Option[String] = None, limit: Option[Int] = None)
+
+/// Dummy class that everything can deserialize to if we don't care about the
+/// output, or if we want serialize `{}`.
+final case class EmptyObj()
```

Comment:
I wonder if we can use `JsonObject.empty` instead directly, did you try it?

Reply:
I've used it as an empty type. For example in 
```kotlin
post[Unit, EmptyObj, Items[JsonObject]](...)
```
It doesn't work if I enter `JsonObject.empty`, because it is not a type, but an instance of `JsonObject`.